crypto/armcap.c

```diff
@@ -150,7 +148,8 @@ void OPENSSL_cpuid_setup(void)
     sigprocmask(SIG_SETMASK, &ill_act.sa_mask, &oset);
     sigaction(SIGILL, &ill_act, &ill_oact);
 
-    if (getauxval != NULL) {
+# ifdef OSSL_IMPLEMENT_GETAUXVAL
```

Comment:
could you move this while block up befor the sigfillset ?
if I see that right, it does not need SIGILL handling any more?

Reply:
of course OPENSSL_armcap_P = 0 need to be move as well.

Reply:
Sure.  Fixup coming up!